modules/cluster-logging-deploy-cli.adoc

```diff
@@ -44,15 +44,8 @@ metadata:
   labels:
     openshift.io/cluster-monitoring: "true" <2>
 ----
-<1> You must specify the `openshift-operators-redhat` Namespace. To prevent
-possible conflicts with metrics, you should configure the Prometheus Cluster
-Monitoring stack to scrape metrics from the `openshift-operators-redhat`
-Namespace and not the `openshift-operators` Namespace. The `openshift-operators`
-Namespace might contain Community Operators, which are untrusted and could publish
-a metric with the same name as an {product-title} metric, which would cause
-conflicts.
-<2> You must specify this label as shown to ensure that cluster monitoring
-scrapes the `openshift-operators-redhat` Namespace.
+<1> You must specify the `openshift-operators-redhat` Namespace. To prevent possible conflicts with metrics, you should configure the Prometheus Cluster Monitoring stack to scrape metrics from the `openshift-operators-redhat` Namespace and not the `openshift-operators` Namespace. The `openshift-operators` Namespace might contain Community Operators, which are untrusted and could publish a metric with the same name as an {product-title} metric, which would cause conflicts.
```

Comment:
Namespace shouldn't be capitalized here (unless it starts the sentence).  And I don't see a need to capitalize "community" either.

 The `openshift-operators` namespace might contain community Operators...

Reply:
Updated. Thanks.